file-formats/tabl/README.md

```diff
@@ -0,0 +1,6 @@
+# TABL File Format
+
+File | Cardinality | Definition | Schema | Example
+:--- | :---  | :--- | :--- | :---
+`<name>.tabl.json` | 1 | *does not exist, yet* | *does not exist, yet* | *does not exist, yet*
+`<name>.tabl.<indexname>.indx.json` | 0..n | [`zif_aff_indx_v1.intf.abap`](./type/zif_aff_indx_v1.intf.abap) | [`indx-v1.json`](./indx-v1.json) | [`zaff_exmpl_tabl.tabl.001.indx.json`](./examples/zaff_exmpl_tabl.tabl.001.indx.json)
```

Comment:
If we don't add an example in the first step, we might have to remove the links.
```suggestion
`<name>.tabl.<indexname>.indx.json` | 0..n | [`zif_aff_indx_v1.intf.abap`](./type/zif_aff_indx_v1.intf.abap) | [`indx-v1.json`](./indx-v1.json) | `zaff_exmpl_tabl.tabl.001.indx.json`
```